model_analyzer/config/generate/perf_analyzer_config_generator.py

```diff
@@ -77,8 +74,8 @@ def __init__(self,
         self._generator_started = False
 
         self._last_results: List[RunConfigMeasurement] = []
-        self._concurrency_results: List[RunConfigMeasurement] = []
-        self._batch_size_results: List[RunConfigMeasurement] = []
+        self._concurrency_results: List[Optional[RunConfigMeasurement]] = []
```

Comment:
Is this still correct? I don't think it is

Reply:
Ye